### PR TITLE
OI4-198

### PR DESCRIPTION
**Pull request by RiccardoScuderi-EH**

### CODE REVIEW

File: packages/oi4-oec-service-node/jest.config.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```
Comment (vdedominicis-EH) [inline packages/oi4-oec-service-node/jest.config.js]: I have no problem with the following instruction, maybe we can remove this disable next line

File: packages/oi4-oec-service-node/src/Proxy/Messagebus/MqttSettings.ts [hunk 2]
```diff
@@ -1,8 +1,17 @@
-export interface MqttSettings {
```
Comment (vdedominicis-EH) [inline packages/oi4-oec-service-node/src/Proxy/Messagebus/MqttSettings.ts:1]: This change is also in the other PR, as discussed double check the changes and the branches.

**Review by vdedominicis-EH — DISMISSED** [review]
Looks good to me, but mybe @Weinschenk wants to review it as well.

File: packages/oi4-oec-service-node/jest.config.js (hunk 1 above)
Comment (RiccardoScuderi-EH) [reply packages/oi4-oec-service-node/jest.config.js]: removed and it workd
s

File: packages/oi4-oec-service-node/src/Proxy/Messagebus/MqttSettings.ts [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
-    useUnsecureBroker?: boolean,
-    username?: string,
-    password: string,
+import {IClientOptions} from 'async-mqtt';
+import os from 'os';
+
+export interface MqttSettings extends IClientOptions{
+    host?: string;
+    port?: number;
+    useUnsecureBroker?: boolean;
```
Comment (Weinschenk) [inline packages/oi4-oec-service-node/src/Proxy/Messagebus/MqttSettings.ts]: Please remove the possibility to use an unsecured broker connection

File: packages/oi4-oec-service-node/tests/connect_via_mqtts.test.ts [hunk 4]
```diff
@@ -0,0 +1,83 @@
+import mqtt = require('async-mqtt');
```
Comment (Weinschenk) [inline packages/oi4-oec-service-node/tests/connect_via_mqtts.test.ts]: Please move tests to src/__tests__ and fixtures to src/__fixtures__

File: packages/oi4-oec-service-node/tests/fixtures/README [hunk 5]
```diff
@@ -0,0 +1,3 @@
+DIE Zertifikate SIND 1 JAHR GÜLTIG.
```
Comment (Weinschenk) [inline packages/oi4-oec-service-node/tests/fixtures/README]: Please move to src/__fixtures__

File: packages/oi4-oec-service-node/tests/fixtures/ca-root-cert.crt [hunk 6]
```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```
Comment (Weinschenk) [inline packages/oi4-oec-service-node/tests/fixtures/ca-root-cert.crt]: Please move to src/__fixtures__

File: packages/oi4-oec-service-node/tests/fixtures/decrypted-client-cert.crt [hunk 7]
```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```
Comment (Weinschenk) [inline packages/oi4-oec-service-node/tests/fixtures/decrypted-client-cert.crt]: Please move to src/__fixtures__

File: packages/oi4-oec-service-node/tests/fixtures/decrypted-client-key.key [hunk 8]
```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```
Comment (Weinschenk) [inline packages/oi4-oec-service-node/tests/fixtures/decrypted-client-key.key]: Please move to src/__fixtures__

File: packages/oi4-oec-service-node/tests/fixtures/encrypted-client-cert.crt [hunk 9]
```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```
Comment (Weinschenk) [inline packages/oi4-oec-service-node/tests/fixtures/encrypted-client-cert.crt]: Please move to src/__fixtures__

File: packages/oi4-oec-service-node/tests/fixtures/encrypted-client-key.key [hunk 10]
```diff
@@ -0,0 +1,30 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----
```
Comment (Weinschenk) [inline packages/oi4-oec-service-node/tests/fixtures/encrypted-client-key.key]: Please move to src/__fixtures__

File: packages/oi4-oec-service-node/tests/fixtures/passphrase.txt [hunk 11]
```diff
@@ -0,0 +1 @@
+abcd
```
Comment (Weinschenk) [inline packages/oi4-oec-service-node/tests/fixtures/passphrase.txt]: Please move to src/__fixtures__

**Review by Weinschenk — CHANGES_REQUESTED** [review]

**Review by Weinschenk — APPROVED** [review]

**Pull request merged**